java/api/src/main/java/org/ray/api/RayActor.java

```diff
@@ -5,12 +5,17 @@
 /**
  * A handle to an actor.
  *
- * @param <T> The type of the concrete actor class.
+ * @param <A> The type of the concrete actor class.
  */
-public interface RayActor<T> {
+public interface RayActor<A> extends ActorCall<A> {
 
   /**
    * @return The id of this actor.
    */
   ActorId getId();
+
+  @Override
+  default RayActor<A> getThis() {
```

Comment:
Can we avoid this? It should not be exposed as an API.

Reply:
yeah, it's a bit weird here. I'll move it to the impl class.